utils/swaggerDefinition.js

```diff
@@ -53,6 +53,43 @@ const swaggerOptions = {
             }
           }
         },
+        tasks: {
+          type: 'object',
+          properties: {
+            type: {
+              type: 'string'
+            },
+            links: {
+              type: 'array',
+              items: []
+            },
+            endsOn: {
+              type: 'string'
+            },
+            startedOn: {
+              type: 'string'
+            },
+            status: {
+              type: 'string'
+            },
+            ownerId: {
+              type: 'string'
+            },
+            percentCompleted: {
+              type: 'number'
+            },
+            dependsOn: {
+              type: 'array',
+              items: []
+            },
+            completionAward: {
+              type: 'object'
```

Comment:
We should be defining the data in these objects too as those keys won't be changing anytime.

Reply:
What to do if we do not know exactly the object keys for this ? like for `completionAward` we can reward the user with only `gold` or with `gold & silver` or `gold, silver, bronze`
in this case should I define the properties as `gold, silver & bronze` ?

Reply:
Yes